sparkler-core/docker-compose.dev.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.9"
```

Comment:
Please ensure all of your files contain an ALv2.0 header. For example see
https://github.com/USCDataScience/sparkler/blob/master/sparkler-core/bin/dockler.sh#L3-L18